tests/playwright/resources/kubernetes/pod-configmap-secret.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-configmap-secret
+spec:
+  containers:
+    - name: app
```

Comment:
I general, since the resources are growing, please use specific naming convention for pods, yamls, etc.
I can imagine here it would be test-app-config.